src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -268,6 +268,7 @@ public void addStudent(Student student) {
         requireNonNull(student);
         Person personMatch = persons.get(student.getName());
         persons.setPerson(personMatch, student);
+        System.out.println("add student: " + persons.get(student.getName()));
```

Comment:
can remove print statement?

Reply:
Thanks for spotting it! Will do the changes!